spec/index.html

```diff
@@ -294,10 +294,9 @@ <h2>Terminology</h2>
     </p>
 
     <p class="note" title="Documents in the stream are independent of each other">
-        Each document in a stream has its own context, [[YAML]] directives, named anchors, and so on.
+        For interoperability considerations on YAML streams,
+        see the YAML Streams section in [[I-D.ietf-httpapi-yaml-mediatypes]].
```

Comment:
```suggestion
        For interoperability considerations on <a>YAML streams<a>,
         see the YAML Streams section in [[I-D.ietf-httpapi-yaml-mediatypes]]
         <div class="ednote">Update to a data-cite reference when the ID has a YAML Streams section</div>.
```

Reply:
This is the link https://github.com/ietf-wg-httpapi/mediatypes/blob/main/draft-ietf-httpapi-yaml-mediatypes.md#yaml-streams-int-yaml-streams but I don't know how to reference it via [[xxx]]

Reply:
That link seems to be to an unpublished draft. The normative reference in our spec is https://datatracker.ietf.org/doc/draft-ietf-httpapi-yaml-mediatypes/. Once the ID is updated with the section on streams, we would change this to `<a data-cite=-D.ietf-httpapi-yaml-mediatypes#section-3.4">YAML Streams</a>`.

Actually, the cited document should probably be the HTML version at https://datatracker.ietf.org/doc/html/draft-ietf-httpapi-yaml-mediatypes-02, rather than the otherwise opaque version we use currently, which exposes the section identifiers, such as `#section-3.1`. So, I'd recommend updating the localBibliography entry to use that URL instead of the current one.

Any time-frame for  a 3rd draft of the ID?

Reply:
This is the localBiblio entry containing the href which should be updated:

https://github.com/json-ld/yaml-ld/blob/d1b039d37a1333276d341f8d341d7b2494741f0b/spec/index.html#L24-L35

Reply:
done